source/sync/error-handling/errors.txt

```diff
@@ -138,6 +138,9 @@ The following errors may occur when your App uses :ref:`Flexible Sync
          does not have write permissions to.
        - Modifying an object in such a way that client would no longer have
          write permissions on that object or field after the write.
+       - Updating an object's :ref:`indexed queryable field <fs-indexed-queryable-fields>` 
+         value from a device results in a compensating write which undoes 
+         the update on the device.
```

Comment:
- Since this is a list of illegal operations and the there is an explanation of what a compensating write is after this section, I think we can drop the second bit here `results in a compensating write which...`
- All compensating write errors only apply to devices so the `from a device` piece is probably not needed since it's mentioned earlier that this applies to clients only
- This also only applies to objects that already exists (you can set the value on creation of an object so it might be nice to make this explicit.)

Thoughts on going with something like this?
```
Updating an existing object's :ref:`indexed queryable field <fs-indexed-queryable-fields>` value
```